config/app.php

```diff
@@ -195,6 +196,7 @@
 		// 'Bugsnag'   => 'Bugsnag\BugsnagLaravel\BugsnagFacade',
 		'Form'      => 'Illuminate\Html\FormFacade',
 		'HTML'      => 'Illuminate\Html\HtmlFacade',
+	    'Gravatar'  => 'Thomaswelton\LaravelGravatar\Facades\Gravatar'
```

Comment:
More mixed spaces and tabs... 

![](http://s2.quickmeme.com/img/46/4675064e74be608035200a63349b1fa6f78b7553b2846c980ce05f55eb4762eb.jpg)